crates/tauri-cli/src/build.rs

```diff
@@ -55,6 +55,10 @@ pub struct Options {
   /// Skip prompting for values
   #[clap(long, env = "CI")]
   pub ci: bool,
+  /// Try to remove unused commands registered from plugins base on the ACL list,
+  /// this feature requires tauri-plugins 2.1 and tauri 2.4
```

Comment:
```suggestion
  /// Try to remove unused commands registered from plugins base on the ACL list,
  /// this feature requires tauri-plugins 2.1 and tauri 2.4.
  /// Alternatively, you can set the environment variable `REMOVE_UNUSED_COMMANDS` to the absolute path of the `src-tauri` directory for the Cargo build,
  /// which is equivalent to enabling this feature for the tauri-cli build.
```

Can we mention the usage of the `REMOVE_UNUSED_COMMANDS` environment variable in the documentation?

Reply:
I'm not sure if we should do it here as this is for the `tauri-cli build --help` output, but I think we can mention it in the docs https://tauri.app/concept/size/ once this feature lands

Reply:
> but I think we can mention it in the docs <https://tauri.app/concept/size/> once this feature lands

I agree with you.

I didn't find other documentation sections in the PR, which led me to mistakenly believe that `tauri-cli build --help` would serve as the documentation. 😂

Reply:
Honestly I feel like this is an implementation detail we should hide from normal users, or at least we don't want people to rely on it as this might change in the future, so probably a collapsed section in the docs that documents how the internal works would be most fitting

Reply:
I think I can accept that. Or, mark it as unstable in the documentation to inform users that "enabling this feature through environment variables" might be removed at any time?

Reply:
I think we just document this in the website as an internal thing, and state something like we did for the other unstable things

> /// # Stability
> /// This is the output of the [`generate_context`] macro, and is not considered part of the stable API.
> /// Unless you know what you are doing and are prepared for this type to have breaking changes, do not create it yourself.

Reply:
LGTM 👍